Mobile_Detect.php

```diff
@@ -741,7 +741,7 @@ public function getHttpHeaders()
      *
      * @return string|null The value of the header.
      */
-    public function getHttpHeader($header)
+    public function getHttpHeader(string $header): string
```

Comment:
In my tests, this function never returned null, but if the creators judge necessary, this can be fix with [PHP 7.1 Nullable Type](http://php.net/manual/en/migration71.new-features.php#migration71.new-features.nullable-types)